lighthouse-core/test/network-records-to-devtools-log.js

```diff
@@ -97,11 +110,11 @@ function getResponseReceivedEvent(networkRecord, index) {
         url: networkRecord.url || exampleUrl,
         status: networkRecord.statusCode || 200,
         headers,
-        mimeType: networkRecord.mimeType || 'text/html',
+        mimeType: typeof networkRecord.mimeType === 'string' ? networkRecord.mimeType : 'text/html',
         connectionReused: networkRecord.connectionReused || false,
         connectionId: networkRecord.connectionId || 140,
-        fromDiskCache: networkRecord.fromDiskCache || undefined,
-        fromServiceWorker: networkRecord.fetchedViaServiceWorker || undefined,
+        fromDiskCache: networkRecord.fromDiskCache || false,
+        fromServiceWorker: networkRecord.fetchedViaServiceWorker || false,
```

Comment:
these two changes are fine, but am I misreading that `networkRequest` should be properly handling either of them being `undefined`?

Reply:
> these two changes are fine, but am I misreading that `networkRequest` should be properly handling either of them being `undefined`?

or is this to match what the protocol is actually sending (`false`) in those events

Reply:
> or is this to match the default that the protocol is actually sending (false) in those events

bingo 👍